xAPI-Communication.md

```diff
@@ -439,8 +446,8 @@ syntax after this represents the particular Resource used. A full list of the re
 described in [Section 6.4.2 OAuth Authorization Scope](#oauthscope).
 * <a name="2.0.s1.b3"></a>The LRS MAY support additional resources not described in this specification. 
 * <a name="2.0.s1.b4"></a>Past, current and future versions of this specification do not and will not define resource endpoints 
-with path segments starting 'extensions/'. LRSs supporting additional resources not defined 
-in this specification SHOULD define their endpoints with path segments starting with 'extensions/'.
+with path segments starting with `extensions/`. LRSs supporting additional resources not defined in this specification SHOULD 
+define their endpoints/resource locations with path segments starting with `extensions/`.
```

Comment:
We should either call it endpoints or resource locations, not both. I prefer the original 'endpoints'.